index.js

```diff
@@ -83,16 +39,17 @@ const renderPagination = (totalPages) => {
     }
     li.addEventListener('click', (event) => {
       event.preventDefault();
-      currentPage = page;
-      renderAllBooks();
-      savePageState(); // Calling the Fn to save the current page state.
+      if (page >= 1 && page <= totalPages) {
+        currentPage = page;
+        renderAllBooks();
+      }
     });
     return li;
   };
 
   paginationElement.appendChild(createPageItem(currentPage - 1, 'Previous'));
 
-  for (let i = 1; i <= totalPages; i++) {
+  for (let i = 1; i <= totalPages; i += 1) {
```

Comment:
why this change? any difference in `i++` and `i += 1`

Reply:
ESLint error: changed to i++